composer/optim/__init__.py

```diff
@@ -1,5 +1,17 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""Optimizers and schedulers.
```

Comment:
[nit] While this is a Google compatible style docstring... bc we put this in our docs page it just appears as a sentence fragment.  Maybe we could just delete, reword the next sentence a bit and move up?  '

Would look like:
```
"""Composer is compatible with optimizers based off of PyTorch's native :class:`~torch.optim.Optimizer` API.

Common optimizers...
"""
```

This would still be Google style compliant and remove the fragment.

Reply:
Hmm, that's not entirely true. The table for the API reference page shows the docstring summary, so we do need a short phrase.

<img width="849" alt="Screen Shot 2022-03-07 at 18 35 41" src="https://user-images.githubusercontent.com/6209880/157155099-b7020d24-5cb6-4cfa-983c-3a24cc379d3a.png">

Reply:
Good point about the docstring summary table.  I would still prefer a complete complete yet concise sentence here.  It would need to serve as a good standalone description for the summary table as well as a good introductory sentence for the doc.

Now that you have me looking at the table I’m actually thinking it could use some cleanup as well.  Some of the descriptions are probably too long (profiler, core?) where as some are not really descriptive (loggers, utils).  Can clean this up in another PR though.

Reply:
Yeah I saw this too and put in a PR to fix: https://github.com/mosaicml/composer/pull/688